### ISSUE TITLE
Performance for multi-label classification is so low?

### ISSUE THREAD

**Issue by XSilverBullet** [google-research/bert#578]
eval_accuracy = 0.05
eval_loss = 0.1684

**Comment by wmarinho-uff** [comment]
I'm having the same problem... I guess that I'm doing something wrong and the gradient vanish... 

The accuracy is random guessing (1/#classes)

**Comment by Nitzg** [comment]
how did you run multi-label classification?
I run the example code in colab and managed to use it to classify my own data for 2 classes, but couldn't extract it to 3 classes (or more, obviously)

thanks!

**Comment by YuMiaoTHU** [comment]
hey, have you solve the problem? I met the same situation，I can't find what's wrong...
